MMM-MonthlyCalendar.js

```diff
@@ -141,7 +141,10 @@ Module.register("MMM-MonthlyCalendar", {
 
           if (self.config.hideDuplicateEvents) {
             self.events = self.events.filter((entry, index, events) => {
-              return (index === 0) || (!equals(entry, events[index - 1]));
+              return (index === 0) ||
+                (!equals(entry.title, events[index - 1].title)
+                  && !equals(parseInt(entry.startDate), parseInt(events[index - 1].startDate))
+                  && !equals(parseInt(entry.endDate), parseInt(events[index - 1].endDate)));
```

Comment:
Are you sure this is right?  I think this will filter out adjacent entries if any of those three fields are equal now, so just having two entries that start or end at the same time will return false here.  Also, since each of these fields is not an array or object, I think it should be `(index === 0) || (entry.title !== events[index - 1].title) || (entry.startDate !== events[index - 1].startDate) || (entry.endDate !== events[index - 1].endDate)`, can you check with that instead?

Reply:
I checked with that logic and nothing was deduped. In the PR I used the logic from the default calendar but then realized it was comparing to the entire entry list rather than just the previous entry, so my logic was faulty as well and leaked through some events. All that's to say, this PR only kind of worked. Oops! 

I am going to go ahead and close this because I modified the `broadcastEvents` function in the default MagicMirror `calendar` module. I set `limitNumberOfEntries` to `true` so that MMM-MonthlyCalendar is receiving already de-duped events, which works for my use case. 

Thanks for the quick response and the gut check and thanks for making this module!